src/main/java/seedu/multitasky/logic/commands/SaveCommand.java

```diff
@@ -4,30 +4,39 @@
 
 // @@author A0132788U
 /**
- * Checks validify of entered filepath and sets the filepath to store entrybook data in a user-defined location.
+ * Checks validity of entered path and saves data at the given file.
  */
-public class SetCommand extends Command {
 
-    public static final String COMMAND_WORD = "set";
+public class SaveCommand extends Command {
 
-    public static final String MESSAGE_SUCCESS = "File path set successfully to ";
+    public static final String COMMAND_WORD = "save";
+
+    public static final String MESSAGE_SUCCESS = "Entrybook data saved successfully to ";
     public static final String MESSAGE_FAILURE = "Invalid file path!\n";
+    public static final String MESSAGE_EXISTS = "File already exists! Rename file.\n";
     public static final String SAMPLE_FILEPATH = " /Users/usernamehere/Desktop/entrybook.xml";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets file path of the entrybook\n" + "Format: "
-                                               + COMMAND_WORD + " newfilepath\n" + "Example: " + COMMAND_WORD
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Saves entrybook data to given filepath\n" + "Format: "
+                                               + COMMAND_WORD + " newfilepath.xml\n" + "Example: " + COMMAND_WORD
                                                + SAMPLE_FILEPATH;
     private final String newFilePath;
 
-    public SetCommand(String newFilePath) {
+    public SaveCommand(String newFilePath) {
         this.newFilePath = newFilePath.trim();
     }
 
+    /**
+     * Executes the Set command if the path is valid and file doesn't already exist in this location.
+     */
     @Override
     public CommandResult execute() {
         if (isValidPath(newFilePath)) {
-            model.changeFilePath(newFilePath);
-            return new CommandResult(MESSAGE_SUCCESS + newFilePath);
+            if ((new File(newFilePath)).isFile()) {
+                return new CommandResult(MESSAGE_EXISTS);
+            } else {
+                model.changeFilePath(newFilePath);
+                return new CommandResult(MESSAGE_SUCCESS + newFilePath);
+            }
         } else {
             return new CommandResult(MESSAGE_FAILURE + MESSAGE_USAGE);
```

Comment:
same as ^.
basically just do a ctrl f for MESSAGE_FAILURE and throw CommandException at those places